src/internal/connector/exchange/mock/mail.go

```diff
@@ -735,13 +736,15 @@ func MessageWithNestedItemAttachmentContact(t *testing.T, nested []byte, subject
 
 func serialize(t *testing.T, item serialization.Parsable) []byte {
 	wtr := kjson.NewJsonSerializationWriter()
+	defer wtr.Close()
```

Comment:
While you're in the func, can I suggest a nit?  `wtr` always reads like `water` to me.
```suggestion
	w := kjson.NewJsonSerializationWriter()
	defer w.Close()
```